assume/units/powerplant.py

```diff
@@ -197,6 +197,9 @@ def calculate_energy_operational_window(
         # adjust max_power if sold positive reserve capacity on control reserve market
         max_power = max_power - self.outputs["capacity_pos"].at[start]
 
+        # ensure min_power < max_power
+        min_power = min(min_power, max(0, max_power))
```

Comment:
under which circumstances does this fail?
I would expect an assertion `assert min_power <= max_power` to fail never?

Reply:
But we add something to min_power and substract from max_power. So it is still possible to have min_power > max_power

Reply:
Is there a nicer formulation or can I leave it like that?